src/plugins/console/public/lib/osd/api.ts

```diff
@@ -29,12 +29,14 @@
  */
 
 import _ from 'lodash';
-import { UrlPatternMatcher } from '../autocomplete/components';
+import { SharedComponent, UrlPatternMatcher } from '../autocomplete/components';
 import { UrlParams } from '../autocomplete/url_params';
 import {
   globalsOnlyAutocompleteComponents,
   compileBodyDescription,
 } from '../autocomplete/body_completer';
+import { Endpoint } from '../autocomplete/types';
```

Comment:
where is this `../autocomplete/types`? did we define it somewhere?

Reply:
Sorry, I forgot to mention that this PR should be merged after https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4148, where this file was created.